queries/vim/highlights.scm

```diff
@@ -151,6 +152,8 @@
 
 ;; Command command
 
+(command) @string
```

Comment:
An alternative could be `@function.call` but I understand the wish to distinguish function calls from command invocations in viml